poorwsgi/session.py

```diff
@@ -202,10 +230,14 @@ def __init__(self, req, expires: int = 0, max_age: int = None,
         self.cookie: SimpleCookie = SimpleCookie()
         self.cookie[SID] = ''
 
-        raw = None
+        if not isinstance(secret_key, (str, bytes)):  # backwards compatibility
+            self.load(secret_key.cookies)
 
-        if req.cookies and SID in req.cookies:
-            raw = req.cookies[SID].value
+    def load(self, cookies: Union[SimpleCookie, tuple]):
+        """Load session from request it's cookie"""
```

Comment:
I'm not sure about the meaning of this sentence. Do you want to say: "Load session from request cookie"?